lib/strapi/projectsGraphQL.js

```diff
@@ -27,9 +27,9 @@ export const fetchStrapiProjects = async (preview = false) => {
     return array.map((project) => ({
       id: project.id,
       name: project.attributes.name,
-      slug: project.attributes.project_id,
+      slug: project.attributes.slug,
       description: project.attributes.description,
-      featuredImage: project.attributes.featured_image.data?.attributes ? project.attributes.featured_image.data.attributes.url : homeHero.src
+      featuredImage: project.attributes.featuredImage.data?.attributes ? project.attributes.featuredImage.data.attributes.url : homeHero.src
```

Comment:
javascript has clean syntax to do this kind of thing. consider this equivalent evaluation:
```
featuredImage: project.attributes.featuredImage.data?.attributes?.url || homeHero.src
```

Reply:
thank you for the suggestion! will try and test this now!